opm/input/eclipse/Python/PyRunModule.cpp

```diff
@@ -81,7 +82,7 @@ bool PyRunModule::executeInnerRunFunction(const std::function<void(const std::st
     } catch(...) {
         auto errorMessage = fmt::format("General exception thrown when calling run(ecl_state, schedule, report_step, summary_state, actionx_callback) function of {}", this->module_name);
         OpmLog::error(errorMessage);
-        throw std::runtime_error(errorMessage);
+        OPM_THROW(std::runtime_error, errorMessage);
```

Comment:
ah I should have been more explicit. The whole point of using OPM_THROW is that it also puts the error in the log files.

Reply:
yes, I just saw that :D and was about to change it